### ISSUE TITLE
Better Documentation to configure settings for Cross Cluster Search.

### ISSUE THREAD

**Issue by nguyenlp** [elastic/elasticsearch#27173]
**Describe the feature**:  With respect to Cross cluster search settings: 
 https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html#cross-cluster-search-settings

Needs to be updated so that it is very clear to the end user that adding a CCS node means adding it to an existing cluster and then having the ability to query other clusters.

There should/needs to be an example architecture section in the documentation, using docker with docker-compose, to illustrate to users many different solutions: 

- from having an all in one node, which is quite common for devs, to having a robust cluster
- with 2 coordination nodes, 3 master nodes, and 3 data nodes
- finally, to having 2 separate clusters with a CCS node to query both clusters

Basically, mimic what big and small customers are doing so that customers and potential users can setup their own reference model architectures to validate the configurations they need to push to each node.

**Comment by javanna** [comment]
hi @nguyenlp , "adding a CCS node means adding it to an existing cluster" is not 100% correct, you could also have a single node cluster with a CCS node only. I do not follow what needs to be clarified. I am also not sure that our reference guide is the right place for in-depth examples like you mentioned, especially with docker. Maybe we should rather publish a blogpost or some other type of content.

**Comment by nguyenlp** [comment]
Hi @javanna , thanks for your inputs.  Yes a blog post with examples would be really helpful for users to reference.  Basically, it would be an additional resource that the users could get more information and knowledge on along with examples.
